### ISSUE TITLE
Loss doesn't drop in the example

### ISSUE THREAD

**Issue by josedolz** [jeonsworld/ViT-pytorch#47]
Hi, thanks for releasing this code.

I have tried to run the CIFAR-10 (as well as CIFAR-100) example, but in both cases the validation (and training) loss do not decrease, and the validation accuracy gets stuck in 0.01. Is there any hyper-parameter that I need to change from the example code? 

Thanks!

![image](https://user-images.githubusercontent.com/5444841/155616702-f9b82910-e510-48cc-8040-07a38b22d955.png)
